src/cmds/play.c

```diff
@@ -83,7 +96,11 @@ static int fd_callback(const void *inputBuffer, void *outputBuffer,
 int main(int argc, char **argv) {
 	int opt;
 	int err;
+#if PLAY_FROM_FLASH
+	uint32_t audio_flash_addr;
```

Comment:
Is this variable really necessary? I think `audio_start_ptr` can be used directly to parse arguments.